.circleci/config.yml

```diff
@@ -3,15 +3,15 @@ version: 2.1
 executors:
   macos:
     macos:
-      xcode: 13.4.1
+      xcode: 15.1.0
```

Comment:
this enables libspatialite 5.1.0 which seems to be needed on intel mac, with 5.0.1 there was a segfault, see https://github.com/valhalla/valhalla/pull/4253#issuecomment-1877853980